contracts/linear/src/epoch_actions.rs

```diff
@@ -204,6 +204,137 @@ impl LiquidStakingContract {
             self.stake_amount_to_settle = 0;
         }
     }
+
+    /// Due to shares calculation and rounding of staking pool contract,
+    /// the amount of staked and unstaked balance might be a little bit
+    /// different than we requested.
+    /// This method is to sync the actual numbers with the validator.
+    pub fn sync_account_balance(&mut self, validator_id: AccountId) {
```

Comment:
how often should `sync_account_balance` be executed? every epoch, for each validator?

Reply:
after stake/unstake action i think